config/CR01R02v4/CMakeBuild.config

```diff
@@ -0,0 +1,148 @@
+########################################################################################################################
```

Comment:
Should this config be committed?

Reply:
We don't have a config for all hardware though, only for each chip.
For each hardware we have a UICR definition for factory images.